cf/domain.py

```diff
@@ -330,6 +330,111 @@ def del_file_location(
 
         return location
 
+    @classmethod
+    def create_regular(cls, x_args, y_args, bounds=True):
+        """
+        Create a new domain with the given longitude and latitude bounds and
+        intervals.
+
+        .. versionadded:: 3.15.1
+
+        :Parameters:
+
+            x_args: `sequence of numbers`
+                A sequence of three numeric values representing the lower and
+                upper bounds of the longitude dimension and the interval between
+                points along the longitude axis.
+
+            y_args: `sequence of numbers`
+                A sequence of three numeric values representing the lower and
+                upper bounds of the latitude dimension and the interval between
+                points along the latitude axis.
```

Comment:
Perhaps reword this to not use the word "bounds", as that could be misleading

"A sequence of three numeric values. The first two values in the sequence represent the coordinate range (see the bounds parameter for details), and the third value represents the cellsize."